meson.build

```diff
@@ -38,6 +38,22 @@ elif (signing_plugin == 'threaded_unless_check_dep')
   signing_plugin = 'threaded'
 endif
 
+onvif_subdir = 'onvif_media_signing'
+onvif_path = meson.source_root() / onvif_subdir
+
+onvif_check = run_command('test', '-d', onvif_path, check: false)
+
+if onvif_check.returncode() == 0
```

Comment:
I don't think what is inside this if-statement works. Have you tried it?

I think you at this point should keep it as simple as possible and skip all the changes in the meson.build. It should compile perfectly anyhow.